### ISSUE TITLE
Running behind a HTTP proxy

### ISSUE THREAD

**Issue by sourcedelica** [conan-io/conan-package-tools#104]
I did some testing of `ConanMultiPackager` locally today at work where there is a firewall.  I am using `CONAN_DOCKER_IMAGE` and it hangs at the `pip install` step.  I can get `pip` to work through the proxy using `pip --proxy=xxx` but not using environment variables.   Is there a way around this?  If not could you add another environment variable that is used as the `--proxy` argument to `pip` (or something more general)?

**Comment by lasote** [comment]
But only for pip? what about the conan downloads?

**Comment by sourcedelica** [comment]
Good point - my particular project doesn't have any requirements so the pip problem was the only one I had seen (so far).

**Comment by lasote** [comment]
Could you configure that machine to connect using the proxy at OS level?

**Comment by sourcedelica** [comment]
Is that possible using Linux?

**Comment by memsharded** [comment]
Yes, definitely, you can configures proxies at the system level, every distro will have its way, here is the ubuntu one, for example: https://help.ubuntu.com/stable/ubuntu-help/net-proxy.html

**Comment by uilianries** [comment]
Hi @sourcedelica !

Did you solve your problem? 

You could configure Docker to use your custom proxy:

https://docs.docker.com/network/proxy/
